source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1198,6 +1199,18 @@ message TransformationSwapCommutableOperands {
 
 }
 
+message TransformationSwapConditionalBranchOperands {
+
+  // Swaps label ids in OpBranchConditional instruction.
+
+  // Descriptor of the instruction to swap operands of.
+  InstructionDescriptor instruction_descriptor = 1;
+
+  // Fresh result id for the OpLogicalNot instruction.
```

Comment:
"... instruction used to invert the guard."

Reply:
Fixed in b08e5a3.